packages/fuel-gauge/src/contract.test.ts

```diff
@@ -1,53 +1,43 @@
 import { ErrorCode, FuelError } from '@fuel-ts/errors';
-import type {
-  TransactionRequestLike,
-  TransactionResponse,
-  JsonAbi,
-  ScriptTransactionRequest,
-  TransferParams,
-} from 'fuels';
 import {
   BN,
   getRandomB256,
   bn,
   multiply,
   toHex,
   toNumber,
-  Provider,
   Contract,
   transactionRequestify,
   Wallet,
   ContractFactory,
-  ZeroBytes32,
-  FUEL_NETWORK_URL,
   Predicate,
   PolicyType,
+  ZeroBytes32,
   buildFunctionResult,
 } from 'fuels';
-import {
-  generateTestWallet,
-  seedTestWallet,
-  expectToThrowFuelError,
-  ASSET_A,
-  ASSET_B,
-} from 'fuels/test-utils';
-
-import { FuelGaugeProjectsEnum, getFuelGaugeForcProject } from '../test/fixtures';
-
-import { createSetupConfig } from './utils';
+import type { JsonAbi, ScriptTransactionRequest, TransferParams } from 'fuels';
+import { expectToThrowFuelError, ASSET_A, ASSET_B, launchTestNode } from 'fuels/test-utils';
 
-const { binHexlified: predicateBytecode } = getFuelGaugeForcProject(
-  FuelGaugeProjectsEnum.PREDICATE_TRUE
-);
-
-const { binHexlified: contractBytecode, abiContents: abi } = getFuelGaugeForcProject(
-  FuelGaugeProjectsEnum.CALL_TEST_CONTRACT
-);
-
-const setupContract = createSetupConfig({
-  contractBytecode,
-  abi,
-});
+import {
+  CallTestContractAbi__factory,
+  StorageTestContractAbi__factory,
+} from '../test/typegen/contracts';
+import CallTestContractAbiHex from '../test/typegen/contracts/CallTestContractAbi.hex';
+import StorageTestContractAbiHex from '../test/typegen/contracts/StorageTestContractAbi.hex';
+import { PredicateTrueAbi__factory } from '../test/typegen/predicates/factories/PredicateTrueAbi__factory';
+
+import { launchTestContract } from './utils';
+
+const contractsConfigs = [
```

Comment:
Please consider using a type notation here.

> cc @nedsalk 


```suggestion
const contractsConfigs: <TypeHere> = [
```

Reply:
This would require a refactor of the types to be feasible because currently it would look like this:
```ts
const contractConfigs: LaunchTestNodeOptions<
  [
    { deployer: typeof CallTestContractAbi__factory; bytecode: string },
    { deployer: typeof CallTestContractAbi__factory; bytecode: string },
  ]
>['contractsConfigs'] = [
  { deployer: CallTestContractAbi__factory, bytecode: '' },
  { deployer: CallTestContractAbi__factory, bytecode: '' },
];
```

Reply:
Yep, far from ideal.

Reply:
I think that after #2708 is merged it's pretty acceptable to not specify the type here because it'd look like this:
```ts
const contractConfigs = [CallTestContractAbi__factory, CallTestContractAbi__factory];
```

Reply:
The only types I think are acceptable not to be informed are native ones unless one _chooses_ not to inform rather than being _unable_ to, like in this case. Either way, it's out of the scope of this PR, so I'll resolve it for now.